views/scss/wizard.scss

```diff
@@ -0,0 +1,66 @@
+///
+/// This program is free software; you can redistribute it and/or
+/// modify it under the terms of the GNU General Public License
+/// as published by the Free Software Foundation; under version 2
+/// of the License (non-upgradable).
+///
+/// This program is distributed in the hope that it will be useful,
+/// but WITHOUT ANY WARRANTY; without even the implied warranty of
+/// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+/// GNU General Public License for more details.
+///
+/// You should have received a copy of the GNU General Public License
+/// along with this program; if not, write to the Free Software
+/// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+/// Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+///
+@import "inc/bootstrap";
+
+.section-container {
+    .content-block {
+        // Because 2 consecutive forms need to be styled as 1
+        // we move certain styles from the form to its parent
+        .multi-form-container {
+            background: $canvas;
+            padding: 30px;
+            border: 1px $uiGeneralContentBorder solid;
+            @include simple-border();
+            min-width: 550px;
+
+            form {
+                background: inherit;
+                padding: 0;
+                border: none;
+
+                .form-toolbar {
+                    .loading-button {
+                        float: right;
+                    }
+                }
+            }
+
+            .form-switch-block {
+                padding-bottom: 30px;
+
+                & > label {
+                    display: inline-block;
+                    width: 35%
+                }
+                .form-switch {
+                    display: inline-block;
+                    width: 64%;
+
+                    label {
+                        display: flex;
```

Comment:
are we sure about this flex grid? did we test in ie11?

Reply:
Just a safety feature to make sure the switch cannot wrap onto 2 lines. I have not checked in IE but I don't believe there are bugs which would break the rendering. We use flex in quite a few places.

Reply:
Ok.